tests/integration/ceph_mgr_test.go

```diff
@@ -221,6 +223,14 @@ func (s *CephMgrSuite) TestStatus() {
 	assert.Equal(s.T(), status, "Backend: rook\nAvailable: Yes")
 }
 
+func bytesInfo(bytesSlice []byte){
```

Comment:
just a nit on the func name
```suggestion
func logBytesInfo(bytesSlice []byte){
```